setup.py

```diff
@@ -40,24 +38,16 @@ class custom_build_ext(build_ext):
 be compiled. No C extensions are essential for gensim to run,
 although they do result in significant speed improvements for some modules.
 %s
-
 Here are some hints for popular operating systems:
-
 If you are seeing this message on Linux you probably need to
 install GCC and/or the Python development package for your
 version of Python.
-
```

Comment:
We want to keep all these paragraph breaks. Why remove them? It makes the text hard to follow.